.github/workflows/php.yml

```diff
@@ -27,13 +33,13 @@ jobs:
           ${{ runner.os }}-php-
 
     - name: Install dependencies
-      run: composer install --prefer-dist --no-progress --no-suggest
+      run: composer install --prefer-dist --no-progress
 
     # Add a test script to composer.json, for instance: "test": "vendor/bin/phpunit"
     # Docs: https://getcomposer.org/doc/articles/scripts.md
 
     - name: Run test suite
-      run: composer run-script test
+      run: composer test
       env: # Set the secrets as environment variables
         EMARSYS_API_USERNAME: ${{ secrets.EMARSYS_API_USERNAME }}
         EMARSYS_API_SECRET: ${{ secrets.EMARSYS_API_SECRET }}
```

Comment:
I'd prefer our integration tests din't use Emarsys directly (I didn't realise it did until now).

Not on my agenda for today, but will probably change it to a mock server of some sort.